frontend/public/components/namespace.jsx

```diff
@@ -275,6 +275,10 @@ class NamespaceDropdown_ extends React.Component {
         defaultBookmarks={defaultBookmarks}
         storageKey={NAMESPACE_LOCAL_STORAGE_KEY}
         shortCut="n" />
+      <div className="co-create-from-yaml co-m-nav-link">
+        <span className="pficon pficon-add-circle-o co-create-from-yaml__add-icon" aria-hidden="true"></span>
+        <a href={formatNamespacedRouteForResource('import', activeNamespace)}>Create from YAML</a>
```

Comment:
We have a different action name here than we show at the top of the page. Should it be "Create from YAML" or "Import YAML"?